source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -83,6 +83,11 @@ export class Version {
         return 0;
     }
 
+    isPreviewFeature(): boolean
```

Comment:
A "version" isn't a "feature", this public method name is odd. It should probably simply be `isPreview` or `isPreviewVersion`.

Reply:
Look like isPreviewVersion is a better name than isPreviewFeature. I avoid isPreview because we have IsPreviewMode or something, and it's easy to cause confusion

Reply:
removed